helpers/index.ts

```diff
@@ -121,6 +123,14 @@ export const installTemplate = async (
     await installTSTemplate(props);
   }
 
+  // write configuration files
+  await writeLoadersConfig(props.root, props.dataSources, props.useLlamaParse);
```

Comment:
TS does not support loaders right now, right? I think we should only generate loaders config for python only for now

Reply:
@leehuwuj thanks good idea!